src/orchard/inspect.clj

```diff
@@ -394,7 +394,7 @@
 (defmethod inspect :array [inspector obj]
   (-> inspector
       (render-labeled-value "Class" (class obj))
-      (render-labeled-value "Count" (alength obj))
+      (render-labeled-value "Count" (java.lang.reflect.Array/getLength obj))
```

Comment:
I guess this might be confusing to someone without a comment explaining why we don't just use `alength`.

Reply:
fair, added a comment.
Again, I would have put this in the line above, but as per your other comment, i've added an inline note.